sw/device/tests/sim_dv/keymgr_key_derivation.c

```diff
@@ -56,6 +57,7 @@ const struct dif_keymgr_state_params kCreatorParams = {
 const struct dif_keymgr_state_params kOwnerIntParams = {
```

Comment:
Does this have a type alias you can use instead of the `struct`-prefixed name?

Reply:
yes, it does. Updated, thanks